Yaab breaks npm run build

```
> coinface@0.1.0 build /tmp/build_82caf179bcada5d8263dc894ba080334/gh
> react-scripts build
Creating an optimized production build...
Failed to compile.
Failed to minify the code from this file: 
 	./node_modules/yaab/yaab.js:2 
Read more here: http://bit.ly/2tRViJ9
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! coinface@0.1.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the coinface@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /app/.npm/_logs/2018-04-04T12_56_47_326Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "redux": "^3.7.2",
     "redux-thunk": "^2.2.0",
     "superstruct": "^0.5.1",
-    "yaab": "0.1.0"
+    "yaab": "^0.2.0"
   },
   "scripts": {
     "start": "react-scripts start",

